.github/wiki/Architecture-Documentation.md

```diff
@@ -0,0 +1,63 @@
+This document outlines the software and hardware requirements for running the infrastructure of the [example app](https://github.com/SanG-github/nimble-devops-ic-web). You can refer to the [[Architecture Diagram]] for a visual overview of the infrastructure components.
+
+## Required Hardware
+
+The infrastructure is hosted on AWS and relies on various AWS services, including ECS with Fargate launch type, RDS, and ElastiCache. Migrating to a different cloud provider is possible, thanks to Terraform's flexibility.
+
+### RDS Instance Type
+
+- Instance Type: `db.t3.micro`
```

Comment:
isn't it `db.t3.medium`? https://github.com/sanG-github/nimble-devops-ic-infrastructure/pull/33/files#diff-e350683dbd845c963c09473e02536035959202976a5bfe47a152c7f88fa7365aR42

Reply:
I forgot to update the document after changing it to the new instance type 😪
It was updated in d5040a1 ✅